### ISSUE TITLE
Mouse pointer should vanish

### ISSUE THREAD

**Issue by hollyhook** [still-scene/t3#200]
If the 2nd render is full screen, it would be nice if the mouse cursor would vanish after a few seconds of inactivity.

**Comment by pixtur** [comment]
Can you describe in more detail what you're trying to accomplish? What is your window layout? 
Do you have multiple displays connected? 
Why not just move the pointed into a corner?

Messing with the point is always a bit tricky with ImGui (because it's rendered natively by Windows. So we should

**Comment by hollyhook** [comment]
I just have the laptop screen. My workflow is

1) run Tooll in full screen
2) open the 2nd render window
3) switch to the main window (alt+tab)
4) change something in graph window 
5) switch to the 2nd render window and watch the result
repeat 3...

I guess this is another issue specific for laptops ... the output window is just too small to get a impression, so I prefer the 2nd render window. But there is always the mouse pointer disturbing a bit. Sure I can train myself to push it away and back again in the graph window. It is just a little anoyance because it happens so frequently

**Comment by pixtur** [comment]
Ah I see! I think that's a very creative work around to an important use-case that should better solved.
A current work around might be to create a layout (in current version this will be Default F10) with only a single output-window and hide UI (with Shift Esc). Sadly this will not remove the mouse pointer or the Output window tools.

We should add a preview mode that can easily be toggled with a hotkey (e.g. CTRL+Space). I renamed this issue accordingly.

**Comment by hollyhook** [comment]
Good idea, but I insist on being annoyed by some ugly white pixels disturbing my creations (aka mouse pointer)

**Comment by pixtur** [comment]
I think this can now be done by toggling focus mode (F12) in fullscreen (F11).
